README.md

```diff
@@ -154,14 +141,9 @@ The below testing environment: a 8-nodes AWS cluster with 1TB data; Spark-3.1.1
 
 Gluten is licensed under [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0).
 
-# 7 Contact
-
-Gluten was initiated by Intel and Kyligence in 2022. Several companies are also actively participating in the development, such as BIGO, Meituan, Alibaba Cloud, NetEase, Baidu, Microsoft, etc. If you are interested in Gluten project, please contact and subscribe below mailing lists for further discussion.
-
-* For community activity: dev@gluten.apache.org
-* For code repository activity: commits@gluten.apache.org
+# 7 Acknowledgements
 
-# 8 Thanks to our contributors
+Gluten was initiated by Intel and Kyligence in 2022. Several companies are also actively participating in the development, such as BIGO, Meituan, Alibaba Cloud, NetEase, Baidu, Microsoft, etc.
```

Comment:
Add IBM here.

@srinivasst would like to add "Google" here?

Reply:
Yes